app/Http/Resources/PollResource.php

```diff
@@ -17,8 +19,11 @@ public function toArray($request)
         return [
             'id' => $this->id,
             'name' => $this->name,
-            'url' => $this->url,
-            'users' => UserResource::collection($this->whenLoaded('users')),
+            'is_admin' => count($this->users) > 0 ? $this->users->find(Auth::id())->pivot->admin : 0,
```

Comment:
A poll (should) always have at least 1 admin user, this check should be removed

Reply:
Assuming you meant "at least 1 user", I agree and fixed in in branch API